### DISCUSSION TITLE
Is it possible to display svg images ?

### DISCUSSION THREAD

**Discussion by loloof64** [slint-ui/slint#849] [General] [answered]
Hi,

I've looked through the examples, but did not find a way to display a svg. Is it possible ?

This what I tried :

Main.rs

```
fn main() {
    MainWindow::new().run();
}
sixtyfps::sixtyfps! {
    MainWindow := Window {
        width: 640px;
        height: 480px;

        Image {
            source: @image-url("images/Chess_ndt45.svg");
            width: 100px;
            height: 100px;
        }
    }
}
```

Cargo.toml

```

[package]
name = "resize_my_knight"
version = "0.1.0"
edition = "2021"


# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
sixtyfps = "0.1.5"
```

This is the Svg image :

[Chess_ndt45.zip](https://github.com/sixtyfpsui/sixtyfps/files/7898684/Chess_ndt45.zip)

The program works, but the image does not appear.

I'm running it on Ubuntu 20.04 64 bit, with Wayland driver.

Regards

**Comment by tronical** [comment, marked as answer]
Hi @loloof64 ! I tried your example and the SVG shows for me.

This is the directory structure I'm using:

```sh
.
├── Cargo.lock
├── Cargo.toml
├── images
│   └── Chess_ndt45.svg
└── src
    └── main.rs

2 directories, 4 files
```

Note that the `images` folder is relative to where `Cargo.toml` is located.

**Reply by loloof64** [reply]
Thank you very much ! It works 😃 

Indeed, I put the images folder inside the src folder, so I could not see anything.
